src/services/balances.js

```diff
@@ -0,0 +1,11 @@
+class Balances {
+  constructor (deps) {
+    this.balances = new Map()
+  }
+
+  handleMoney (amount) {
+    // TODO: Implement balance logic
```

Comment:
This should be implemented ASAP after this PR is merged

Reply:
Yeah. I think that's a job for @sharafian - he's written that code about 15 times already since it used to be in each plugin.